src/utils/rfqm_health_check.ts

```diff
@@ -1,8 +1,18 @@
+import { BigNumber } from '@0x/utils';
 import { Producer } from 'sqs-producer';
 
-const SQS_QUEUE_SIZE_DEGRADED_THRESHOLD = 10; // More messages sitting in queue than this will cause a DEGRADED issue
-const SQS_QUEUE_SIZE_FAILED_THRESHOLD = 20; // More messages sitting in queue than this will cause a FAILED issue
+import { ETH_DECIMALS } from '../constants';
+import { RfqmWorkerHeartbeatEntity } from '../entities';
 
+const SQS_QUEUE_SIZE_DEGRADED_THRESHOLD = 10; // (minutes) More messages sitting in queue than this will cause a DEGRADED issue
+const SQS_QUEUE_SIZE_FAILED_THRESHOLD = 20; // (minuteds) More messages sitting in queue than this will cause a FAILED issue
+
+const RECENT_HEARTBEAT_AGE_THRESHOLD = 5; // (minutes) Heartbeats older than this will produce a DEGRADED issue. A FAILED issue is produced if NO heartbeats are newer than this.
+
+const BALANCE_FAILED_THRESHOLD = 0.04; // (eth) If NO worker has a balance higher than this, a FAILED issue gets created.
```

Comment:
I would set this higher--like 0.1

Reply:
Me and daniel talked this over and came up with this.. anything below the threshold cuts off RFQM, so as long as there is enough for one tx we shouldn't go FAILED

Reply:
if we're not in a `FAILED` state we could send multiple folks to queue and not all could get mined